pkg/mimirtool/client/alerts.go

```diff
@@ -15,7 +15,7 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-const alertmanagerAPIPath = "/api/v1/alerts"
+const alertmanagerAPIPath = "/api/v2/alerts"
```

Comment:
The only APIs that moved to V2 are the ones inside prometheus alertmanager itself - i.e. the ones hosted under the `/alertmanager` subpath.

Mimir's APIs for managing alertmanager configuration (`/api/v1/alerts`, different `/alertmanager/api/v1/alerts` confusingly) are not deprecated and are still V1. So, `/api/v1/alerts` is still correct - I think this may cause mimirtool to try to also edit configuration against endpoints that don't exist.